### ISSUE TITLE
UnicodeDecodeError when build FW

### ISSUE THREAD

**Issue by Nickolik1** [OpenAstroTech/OATFWGUI#45]
I have successfully downloaded the version, but when I click "Build FW", I get an error:
```
24-02-28 12:06:03,600:ERROR:In file included from src\../LocalConfiguration.hpp:20:0, 
2024-02-28 12:06:03,601:CRITICAL:Exception caught 
2024-02-28 12:06:03,609:CRITICAL:Traceback (most recent call last):
  File "C:\adb\OATF\OATFWGUI\external_processes.py", line 65, in handle_stderr
    stderr = bytes(data).decode("utf8")
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xee in position 176: invalid continuation byte 
2024-02-28 12:06:03,610:CRITICAL:
...
```
Win 10 
 [oat_fw_gui_2024-02-28-12-01-49.log](https://github.com/OpenAstroTech/OATFWGUI/files/14431469/oat_fw_gui_2024-02-28-12-01-49.log)

**Comment by julianneswinoga** [comment]
Thanks for the bug report!! Seems like the process output isn't UTF-8. What language is Windows set to by chance?

**Comment by julianneswinoga** [comment]
I released a new version `1.0.7` which should fix this :slightly_smiling_face:
